api/rocjpeg.h

```diff
@@ -26,6 +26,7 @@ THE SOFTWARE.
 
 #pragma once
 #include "hip/hip_runtime.h"
+#include "rocjpeg/rocjpeg.h"
```

Comment:
why is this line added here? you have included the same rocjpeg/rocjpeg.h inside the header itself. Please remove this line.